extensions/amp-selector/1.0/README.md

```diff
@@ -0,0 +1,287 @@
+# Bento Selector
+
+## Usage
+
+The Bento Selector is a control that presents a list of options and lets the user choose one or many options; the contents of the options aren't just limited to text. It can be used as a web component[`<bento-selector>`](#web-component), or as a Preact/React functional component [`<BentoSelector>`](#preactreact-component).
+
+### Web Component
+
+You must include each Bento component's required CSS library before adding custom styles in order to guarantee proper loading. Or use the lightweight pre-uprgrade styles available inline. See [Layout and Style](#layout-and-style)
+
+#### Example: Import via npm
+
+[example preview="top-frame" playground="false"]
+
+Install via npm:
+
+```sh
+npm install @ampproject/bento-selector
+```
+
+```javascript
+import '@ampproject/bento-selector';
+```
+
+[/example]
+
+#### Example: Include via `<script>`
+
+[example preview="top-frame" playground="false"]
+
+```html
+<head>
+  <script src="https://cdn.ampproject.org/custom-elements-polyfill.js"></script>
+  <!-- These styles prevent Cumulative Layout Shift on the unupgraded custom element -->
+  <style data-bento-boilerplate>
+    bento-selector {
+      display: block;
+      overflow: hidden;
+      position: relative;
+    }
```

Comment:
bento-selector is a container element so we don't want to hide its children and it should not overflow
```suggestion
    bento-selector {
      display: block;
    }
```